res/css/style.css

```diff
@@ -586,3 +586,16 @@ body,
   /* Invert the colors of the checkmark when selected */
   border-color: #fff;
 }
+
```

Comment:
I feel like `res/css/style.css` should eventually be removed, is there somewhere else this rule could go? Even a new file.

Reply:
ah, I had the same feeling, but here I couldn't find a good idea because this is reasonably global. Maybe a global.css or shared.css or utils.css ?

Reply:
I used global.css